relay-dynamic-config/src/global.rs

```diff
@@ -58,11 +66,29 @@ pub struct Options {
     #[serde(rename = "profiling.profile_metrics.unsampled_profiles.enabled")]
     pub unsampled_profiles_enabled: bool,
 
+    /// Kill switch for controlling the cardinality limiter.
+    #[serde(rename = "relay.cardinality-limiter.mode")]
+    pub cardinality_limiter_mode: CardinalityLimiterMode,
+
     /// All other unknown options.
     #[serde(flatten)]
     other: HashMap<String, Value>,
 }
 
+/// Kill switch for controlling the cardinality limiter.
+#[derive(Default, Clone, Copy, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub enum CardinalityLimiterMode {
+    /// Cardinality limiter is enabled.
+    #[default]
+    Enabled,
+    /// Cardinality limiter is enabled but no actions are taken.
```

Comment:
What does "no actions are taken" mean? Could we elaborate that a bit?

Reply:
Rephrased to:

> Cardinality limiter is enabled but cardinality limits are not enforced.

Is that better to understand?

Reply:
Yep, thanks!